pkg/controller/userreport/controller.go

```diff
@@ -55,11 +57,16 @@ func New(cacher cache.Cacher, cfg *config.RedirectConfig, db *database.Database,
 
 	localCache, _ := memcache.New(30 * time.Second)
 
+	httpClient := &http.Client{
+		Timeout: 10 * time.Second,
```

Comment:
We should document this in the webhook docs - if your processor doesn’t respond in 10 seconds, we will error out to the user.

Reply:
Done